File: SocialConnect/src/main/java/com/example/SocialConnect/dto/post/PostResponse.java
package com.example.SocialConnect.dto.post;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
public class PostResponse {
    private Long id;
    private String title;
    private String content;
    private String username;
    private String groupName;
    private LocalDateTime createDate;
    private long likeCount;
    private long loveCount;
    private long dislikeCount;
}

File: SocialConnect-Frontend/src/app/model/PostResponse.ts
import { CommentResponse } from "./CommentResponse";

export interface PostResponse {
  id: number;
  title: string;
  content: string;
  username: string;
  groupName: string;
  createDate: string;
  likeCount: number;
  loveCount: number;
  dislikeCount: number;
  comments: CommentResponse[]; 
}
